l1-contracts/test/portals/TokenPortal.sol

```diff
@@ -22,11 +22,7 @@ contract TokenPortal {
   );
 
   event DepositToAztecPrivate(
-    bytes32 secretHashForRedeemingMintedNotes,
```

Comment:
This ^ was the hash in the transparent note. It's no longer there (and neither is L2 address) meaning that only the message secret "owns" the bridged tokens.